onnxruntime/python/tools/transformers/profile_result_processor.py

```diff
@@ -0,0 +1,354 @@
+import argparse
+import json
+
+"""
+This profiler result processor print out the kernel time spent on each Node of the model.
+Example of importing profile result file from onnxruntime_perf_test:
+    python profile_result_processor.py --input profile_2021-10-25_12-02-41.json
+"""
+
+NODES_TYPE_CONTAINING_SUBGRAPH = ["Scan", "Loop", "If"]
```

Comment:
```suggestion
"""This profiler result processor print out the kernel time spent on each Node of the model.

Example of importing profile result file from onnxruntime_perf_test:
    python profile_result_processor.py --input profile_2021-10-25_12-02-41.json
"""

import argparse
import json

_NODES_TYPE_CONTAINING_SUBGRAPH = frozenset(("Scan", "Loop", "If"))
```

nits:
1. The module docstring should be the first lines. You may also want to add copyright notice.
2. Prepend `_` for private constants
3. Use a frozenset for quick lookup.